promscale/backup-restore.md

```diff
@@ -17,7 +17,11 @@ For example:
 ```bash
 pg_dump -h localhost -p 5432 -U tsdbadmin -d mydb -F p -f dump.sql
 ```
+To backup larger databases use the custom format:
 
+```bash
+pg_dump -h localhost -p 5432 -U tsdbadmin -d mydb -F c -f dump.sql
```

Comment:
don't use the `.sql` extension since it is a binary file. I suggest `dump` or `dump.dat`